language/stdlib/modules/gas_schedule.move

```diff
@@ -12,36 +12,14 @@ address 0x0:
 //    address, and will preload the vector with the gas schedule for instructions. The VM will then
 //    load this into memory at the startup of each block.
 module GasSchedule {
```

Comment:
Should this be merged with `LibraVMConfig`? It doesn't have any utility outside of that module and it would be nice to make `new` private

Reply:
Agreed. I think merging these all into `LibrVMConfig` makes sense.